### DISCUSSION TITLE
What is the maximum of photo download?

### DISCUSSION THREAD

**Discussion by ebattistini** [icloud-photos-downloader/icloud_photos_downloader#487] [Q&A] [answered]
I'm asking the question because I have a library of 46000 photos and 4000 videos on iCloud and currently running icloudpd in a container and my download only shows 16700.

**Comment by menkej** [comment]
I'm currently not aware of a limit in icloudpd. As it relies on the apple infrastructure, there might be a limit on downloads per session. Have you tried a re-run?

**Reply by ebattistini** [reply]
thank you for your answer. My download was running over night so I'm going to do a new run today.

**Comment by ebattistini** [comment, marked as answer]
So there is a new behaviour introduced in Ventura and iOS16. which is shared library. I had that one active when I started the download and the script defaulted to my personal library which was around 16K pictures. 

Now I've removed that option and launch icloudpd I have the total count of 50K images and videos, Just hoping it will not create duplicates...
